olxcleaner/loader/xml_exceptions.py

```diff
@@ -69,7 +69,7 @@ class InvalidPointer(CourseError):
 
```

Comment:
Little concerned about the docstring here. Was `InvalidPointer` used anywhere else? This error description is now completely different to the below description.